rosidl_parser/rosidl_parser/parser.py

```diff
@@ -235,6 +235,7 @@ def extract_content_from_ast(tree):
                 name=goal_request_basename),
             goal_request, result_response, feedback_message)
         content.elements.append(action)
+        content.elements.extend(action.includes)
```

Comment:
This could add an explicit `Include` instead.

Reply:
Sure, that's the other option, assuming we keep both parser and `Action` specific fields in sync.